index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="index.css">
+    </head>
+    <body>
+        <header class="wrapper">
+            <div class="box a">LOGO</div>
+            <div class="box b">
```

Comment:
you can get rid of this div, and just give all necessary classes to `nav`. Extra wrapper is not needed here.